src/main/java/org/sqlite/jdbc3/JDBC3DatabaseMetaData.java

```diff
@@ -681,7 +681,7 @@ public boolean supportsSavepoints() {
 
     /** @see java.sql.DatabaseMetaData#supportsSchemasInDataManipulation() */
     public boolean supportsSchemasInDataManipulation() {
-        return false;
+        return true;
```

Comment:
is this related to the change of this PR, or something we missed ? Database metadata is not DML, thus your changes should not change this value IMHO.

Reply:
agree